server/pages/index.jade

```diff
@@ -58,6 +63,39 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 
 			if sectionCss
 				link(rel='stylesheet', id='section-css', href=sectionCss)
+		
+		//- Preconnect to our CDN.
+		link(rel='preconnect' href='//s0.wp.com')
+		
+		//- Preconnect to our API.
+		link(rel='preconnect' href='https://public-api.wordpress.com')
+		
+		//- Preconnect for analytics scripts.
+		link(rel='preconnect' href='//stats.wp.com')
+		link(rel='preconnect' href='https://www.google-analytics.com')
+		
+		//- Preconnect for Google domains - especially useful for SSO.
+		link(rel='preconnect' href='https://google.com')
+		link(rel='preconnect' href='https://ssl.gstatic.com')
+		link(rel='preconnect' href='https://apis.google.com')
+		link(rel='preconnect' href='https://accounts.google.com')
+		link(rel='preconnect' href='https://stats.g.doubleclick.net')
+
+		//- Preload our scripts
+		if i18nLocaleScript
```

Comment:
Sidenote: if this technique turns out to be useful, we probably should keep the lists for `preconnect` and `preload` in JS file and iterate over them in here. We can think how to do it later :)

Reply:
See in https://github.com/Automattic/wp-calypso/blob/master/server/pages/index.jade#L159 how good assumption I made? 😂

Reply:
That was fast...